src/python/pants/backend/native/subsystems/xcode_cli_tools.py

```diff
@@ -26,75 +24,160 @@ class XCodeCLITools(Subsystem):
 
   options_scope = 'xcode-cli-tools'
 
-  _REQUIRED_TOOLS = frozenset(['cc', 'c++', 'clang', 'clang++', 'ld', 'lipo'])
+  REQUIRED_FILES = {
```

Comment:
Are these intentionally public? Would be good to encapsulate instead if possible, to avoid needing to break APIs to add more data.

Reply:
Yeah, there's no reason for these to be public and the reason you mentioned is a fantastic reason to keep it private.